torch_geometric/graphgym/config.py

```diff
@@ -473,16 +473,21 @@ def assert_cfg(cfg):
     cfg.run_dir = cfg.out_dir
 
 
-def dump_cfg(cfg):
+def dump_cfg(cfg, fname):
```

Comment:
Thanks for you suggestions here. I think `dump_cfg` does not need to be changed. The purpose is to save a copy of cfg to the results folder